<ui version="4.0" >
 <class>ReceiverInputWidget</class>
 <widget class="QWidget" name="ReceiverInputWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QRadioButton" name="radioButtonRSCI" >
     <property name="text" >
      <string>Receiver Status &amp; Control Interface (RSCI)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QRadioButton" name="radioButtonLocal" >
     <property name="text" >
      <string>Hamlib Rig / Soundcard</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QStackedWidget" name="stackedWidgetLR" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="rsci" >
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBoxStatus" >
         <property name="title" >
          <string>Status</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelgroup" >
            <property name="text" >
             <string>Multicast Group</string>
            </property>
            <property name="buddy" >
             <cstring>lineEditGroup</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEditGroup" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelsport" >
            <property name="text" >
             <string>Port</string>
            </property>
            <property name="buddy" >
             <cstring>lineEditsport</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEditsport" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBoxControl" >
         <property name="title" >
          <string>Control</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelHost" >
            <property name="text" >
             <string>Host</string>
            </property>
            <property name="buddy" >
             <cstring>lineEditHost</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEditHost" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelcport" >
            <property name="text" >
             <string>Port</string>
            </property>
            <property name="buddy" >
             <cstring>lineEditcport</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEditcport" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hamlib_sound" >
      <layout class="QGridLayout" name="gridLayout_5" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelRig" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Rig</string>
         </property>
         <property name="buddy" >
          <cstring>comboBoxRig</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QComboBox" name="comboBoxRig" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelSound" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Sound Input</string>
         </property>
         <property name="buddy" >
          <cstring>comboBoxSound</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QComboBox" name="comboBoxSound" />
       </item>
       <item row="2" column="0" colspan="3" >
        <widget class="QFrame" name="soundInputFrame" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radioButtonReal" >
            <property name="text" >
             <string>Real</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="radioButtonIQ" >
            <property name="text" >
             <string>I/Q</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QCheckBox" name="CheckBoxFlipSpec" >
            <property name="text" >
             <string>&amp;Flip Input Spectrum</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QStackedWidget" name="stackedWidgetip" >
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <widget class="QWidget" name="pagereal" >
             <layout class="QGridLayout" name="gridLayout" >
              <item row="0" column="0" >
               <widget class="QRadioButton" name="radioButtonLeft" >
                <property name="text" >
                 <string>Left channel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QRadioButton" name="radioButtonRight" >
                <property name="text" >
                 <string>Right channel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" >
               <widget class="QRadioButton" name="radioButtonMix" >
                <property name="text" >
                 <string>Mix</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageiq" >
             <layout class="QGridLayout" name="gridLayout_4" >
              <item row="0" column="0" >
               <widget class="QRadioButton" name="radioButtonIQPos" >
                <property name="text" >
                 <string>Positive</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QRadioButton" name="radioButtonIQNeg" >
                <property name="text" >
                 <string>Negative</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" >
               <widget class="QCheckBox" name="checkBoxzif" >
                <property name="text" >
                 <string>Zero I/F</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
